dist/tools/testrunner/testrunner.py

```diff
@@ -38,9 +39,15 @@ def run(testfunc, timeout=10, echo=True, traceback=False):
         if traceback:
             print_tb(sys.exc_info()[2])
         return 1
+    except EOF:
```

Comment:
Doesn't work. You really have to use the exception provided by the pexpect. Just add:
```python
from pexpect import exceptions
```
at the beginning of the file and replace this line by:
```python
except exceptions.EOF:
```

Reply:
what about importing `EOF`? You last pep8 fix removed it, because it was unused. I have no strong opinion about this, but we already do the same for `TIMEOUT`

Reply:
> what about importing EOF?

Yes, it's a better solution